packages/client/components/GitHubScopingSearchInput.tsx

```diff
@@ -51,17 +51,22 @@ const GitHubScopingSearchInput = (props: Props) => {
   const {queryString} = githubSearchQuery
   const isEmpty = !queryString
   const atmosphere = useAtmosphere()
-  const placeholder = 'Search issues on GitHub'
+  useEffect(() => {
+    const defaultInput = 'is:issue is:open sort:updated involves:@me '
+    setSearch(atmosphere, meetingId, defaultInput)
+  }, [])
+
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const {value} = e.target
     setSearch(atmosphere, meetingId, value)
   }
   const clearSearch = () => {
     setSearch(atmosphere, meetingId, '')
   }
+
   return (
     <Wrapper>
-      <SearchInput value={queryString} placeholder={placeholder} onChange={onChange} />
+      <SearchInput autoFocus value={`is:issue${queryString.slice(8)}`} onChange={onChange} />
```

Comment:
The GH GraphQL query requires a search type of issue, repo or user, so I'm using `type:ISSUE`. If the user replaced `is:issue` in the query string with `is:user` while we're using `type:ISSUE`, the results won't be very helpful. We could parse the string and update the search type but rating users or repos is an unlikely use case, so I think we should stick with `type:ISSUE`. 

I think we could either:
1. Always show `is:issue` in the search input and prevent the user from deleting it
2. Let the user remove `is:issue` and write `is:user`, returning unhelpful results

I prefer option 1. It might be a bit annoying that the user can't remove the text but always showing `is:issue` acts as a placeholder and makes it more likely that the results are useful

Reply:
this is a fun find!
one problem is what if they change `is:issue` to `is:pr`? while it's probably not a best practice, i imagine some folks would say "let's scope how much effort it takes to finish this PR"

in that case, we probably want to let them edit `is:issue`. but, like you say, we probably don't want to let them shoot themselves in the foot by searching for users in the issues search.

some ideas:
- [ ] we validate the query string & search for `is:user`. if we find it, we remove it
- [ ] we pop a toast asking them why they want to see the world burn
- [ ] if the search returns 0 results, we see if the have `is:user` if they do, instead of saying "No results found" we say "Why are you searching for users you dingbat"

Reply:
i don't much like the first option because we're doing something magical without explaining why (even though it should be obvious). After enough failed credit card & phone number forms that add parenthesis or remove letters, i've learned editing input fields is hard to get right.
 
i rather like the 3rd option, but it's your call!

Reply:
>  we pop a toast asking them why they want to see the world burn
🤣 

I like the third option too - it's less intrusive and works well with our existing UI. Updated!